lib/kamal/configuration.rb

```diff
@@ -20,6 +20,8 @@ class Kamal::Configuration
 
   class << self
     def create_from(config_file:, destination: nil, version: nil)
+      ENV["KAMAL_DESTINATION"] = destination
```

Comment:
Is there a preferred way to set this? I know most config is loaded via the `env` helper, just seeing `VERBOSE` set directly with `ENV`. It also needs to be set as early as possible and `create_from` is the entry point basically.

Reply:
Here is fine I think 👍